src/tmx/TmxUtils/src/kafka/kafka_consumer_worker.cpp

```diff
@@ -42,6 +42,12 @@ namespace tmx::utils
             return false;
         }
 
+        if (conf->set(ENABLE_AUTO_COMMIT, "true", errstr) != RdKafka::Conf::CONF_OK)
```

Comment:
I can look at documentation as well but just curious is you know what additional configuration this is setting?